pwa-shell/README.md

```diff
@@ -20,6 +20,26 @@ the client id used for oauth2 authentication.
 
 the client secret used for oauth2 authentication.
 
+### REACT_APP_AUTH_TYPE
```

Comment:
As far as I see in the code, the default auth type is `default` and not `keycloak`, as also the names itself suggests :)

Reply:
so if we make this default, we should remove the preconfigured REACT_APP_AUTH_TYPE in .env file. is that okay?

Reply:
anyway I have rephrased the statement. it should be "preconfigured" not "default"

Reply:
we can preconfigure the app to default auth type when the time comes we switch the repo with the entando brand app

Reply:
@gidesan if we remove the .env file form the repo which is the alternative to generate static resources based on the env vars?

Reply:
Sorry guys, I meant not to remove .env file (because it still containse `REACT_APP_DOMAIN`) but to remove `REACT_APP_AUTH_TYPE`, `REACT_APP_KEYCLOAK_URL`, `REACT_APP_KEYCLOAK_REALM`, `REACT_APP_KEYCLOAK_CLIENT_ID` from `.env`.

They are not needed for our default auth, but 3rd party devs can read the docs and understand they can configure KC through those vars.

Reply:
Hi @gidesan, if its okay, I can preconfigure to default auth but for the entando brand app only. I can apply this at PWA-12 (https://github.com/entando/entando-pwa/pull/80). For SME though I think we should leave it as it is since they do use keycloak as their authentication.

Reply:
OK, fine for me.